lib/build.js

```diff
@@ -74,8 +74,8 @@ function webpackConfig(dir, additionalConfig) {
     devtool: false
   };
   fs.readdirSync(dirPath).forEach(function(file) {
-    if (file.match(/\.js$/)) {
-      var name = file.replace(/\.js$/, "");
+    if (file.match(/\.(m?js|ts)$/)) {
+      var name = file.replace(/\.(m?js|ts)$/, "");
       webpackConfig.entry[name] = "./" + name;
```

Comment:
One fix needed: above line should be changed to `webpackConfig.entry[name] = "./" + file;` since webpack won't pick non-js files by "name", need explicit extension.